example/android/gradle.properties

```diff
@@ -22,6 +22,7 @@
 
 # Enable dedicated thread pool executor
 AsyncStorage_dedicatedExecutor=true
+AsyncStorage_useNextStorage=true
```

Comment:
Should this be flippable so we can test both paths on CI?

Reply:
on Example app, by default this will be true (so one can develop next version), while our E2E apk build job will turn it off via command line